content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Group Export | R4 API
+---
+
+# Group Export
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+The Group Export operation is used to obtain FHIR resources for all members of a specified group of patients.
+
+An export process uses the [FHIR Asynchronous Request Pattern](https://hl7.org/fhir/R4/async.html) to kick off the export job for a group of patients. Once the export job is kicked off, the status endpoint may be called to check the progress of the job. Once the job is complete, the status endpoint returns a list of files which may be downloaded using the file retrieval endpoint. After the appropriate ndjson files are downloaded, the delete endpoint should be called to clean up the no longer needed files.
+
+The Group Export operation exports resources required for [United States Core Data for Interoperability V1 (USCDI)](https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi). For a full list of resources see the [US Core Profiles](http://hl7.org/fhir/us/core/STU4/#us-core-profiles). When the `_type` parameter is used it returns only the FHIR resources specified. The `_type` parameter should be used to filter the resources required for a specific use case. Limiting the number of requested resources in the `_type` parameter is recommended as it can decrease response times.
```

Comment:
is the http://hl7.org/fhir/us/core/STU4/#us-core-profiles the right link? as there are resources like Medication and PractitionerRole we don't support?

Reply:
> is the http://hl7.org/fhir/us/core/STU4/#us-core-profiles the right link? as there are resources like Medication and PractitionerRole we don't support?

We are certifying on 4.0.0 so not sure why PractitionerRole is in there. We don't support Medication resource itself but it is a valid profile in 4.0.0. Perhaps this would be a better link to STU4 which shows a direct link to the USCDI v1 data element to US Core Profile http://hl7.org/fhir/us/core/STU4/general-guidance.html#us-core-data-for-interoperability

Reply:
Fran the link you have seems more accurate for me.

Reply:
I have updated the link to http://hl7.org/fhir/us/core/STU4/general-guidance.html#us-core-data-for-interoperability: https://github.com/cerner/fhir.cerner.com/pull/899/commits/02316ffb42713ea398525b7522d9996c9f2d84fd